docs/personalization/api_reference/tracking_api.md

```diff
@@ -400,36 +330,30 @@ A predefined rating can be submitted when the user comments on an item.
 ### Wishlist event
 
 This event is issued when the end user puts a product on their shopping list.
-It means that they are interested in the product but plan to buy it later.
+It means that they're interested in the product but plan to buy it later.
 This information can help return better recommendations.
 It can also be used as a basis for the price drop trigger.
 
 `GET https://event.perso.ibexa.co/api/[customerid]/wishlist/[userid]/[itemtypeid]/[itemid]?price=2.50EUR`
 
 For a detailed description of embedded parameters, see [event parameters](#event-parameters).
-In addition to the fact that an item is placed on the shopping list, this event can provide information
-about the product price at the time it was saved.
+In addition to the fact that an item is placed on the shopping list, this event can provide information about the product price at the time it was saved.
 
 |Name|Description|
 |---|---|
 |`price`|A price for a single product. It contains the price in decimal format and the currency code. If the price has a decimal part, the point must be used. There can be no space between price and currency. This parameter is optional.|
 
 ## Tracking events based on recommendations
 
-Tracking events based on integrated recommendations are the only way to measure the accuracy
-and effectiveness of recommendations.
+Tracking events based on integrated recommendations are the only way to measure the accuracy and effectiveness of recommendations.
 Both recommendation response and trigger message include requests to generate these events.
-Events of this type inform the Personalization server about the recommendations that
-were shown to the user and which of those recommendations were clicked.
-Otherwise, it would be impossible to calculate reliable statistics that could be checked
-against the customer's KPIs.
+Events of this type inform the Personalization server about the recommendations that were shown to the user and which of those recommendations were clicked.
+Otherwise, it would be impossible to calculate reliable statistics that could be checked against the customer's KPIs.
 
 A recommendation response includes requests to generate a Rendered and Clickrecommended event.
```

Comment:
According to the [doc code sample](https://ez-systems-developer-documentation--2506.com.readthedocs.build/en/2506/personalization/api_reference/tracking_api/#tracking-events-based-on-recommendations), or [this test fixture](https://github.com/ibexa/personalization/blob/v4.6.13/tests/fixtures/recommendations/response-body.json#L13-L14):
```suggestion
A recommendation response includes requests to generate `rendered` and `clickRecommended` events.
```
It might be something to keep for latter with perso developers to review it.